src/app/containers/IndexPageSection/index.jsx

```diff
@@ -197,14 +197,6 @@ const IndexPageSection = ({
   // This may change in the future, if a way to avoid breaking UX is found.
   // Also, don't render a section without any items.
 
-  // If renderWithoutStrapline is false (which it is by default),
-  // then the section will not render if there is no strapline.
-  // If renderWithoutStrapline is true, and there is no strapline present,
-  // the section will always render.
-  if (!strapline && !renderWithoutStrapline) {
```

Comment:
I think we still need to keep the `!strapline` logic here to maintain backwards compatibility with Home & IDX pages

Reply:
Forgot about that, putting back in.